x/axelarcork/module.go

```diff
@@ -0,0 +1,175 @@
+package axelarcork
+
+import (
+	"context"
+	"encoding/json"
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	sim "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/peggyjv/sommelier/v6/x/axelarcork/client/cli"
+	"github.com/peggyjv/sommelier/v6/x/axelarcork/keeper"
+	"github.com/peggyjv/sommelier/v6/x/axelarcork/types"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+var (
+	_ module.AppModule           = AppModule{}
+	_ module.AppModuleBasic      = AppModuleBasic{}
+	_ module.AppModuleSimulation = AppModule{}
+)
+
+// AppModuleBasic defines the basic application module used by the cork module.
+type AppModuleBasic struct{}
```

Comment:
Shouldn't there be references to the other keepers somewhere in `module.go`?

Reply:
My assumption based on all of the previous modules and SDK modules was that external keepers that are expected to be referenced by a module's keeper should be passed in when creating the app module.

Reply:
Can you show me that? Because I am primarily seeing that modules send some or none of their non-native keepers to NewAppModule